plugins/scaffolder/src/next/TemplateEditorPage/TemplateEditor.tsx

```diff
@@ -57,7 +57,7 @@ const useStyles = makeStyles({
 
 export const TemplateEditor = (props: {
   directory: TemplateDirectoryAccess;
-  fieldExtensions?: NextFieldExtensionOptions<any, any>[];
+  fieldExtensions?: FieldExtensionOptions<any, any>[];
```

Comment:
This type should be reviewed at some stage.  I can't find any cases where the type arguments are not `<any, any>` in the codebase

Reply:
Yeah idd, I tried to convert these to unknown but a lot of stuff broke, maybe something for scaffolder v2 :eyes:

Reply:
should they even be generic arguments if they are set to `<any, any>` or `<unknown, unknown>` every time?

Reply:
The type parameters are important for the type inference of `createScaffolderFieldExtension`. The reason you don't see many other usages is that we generally don't know the exact types of the field extensions that we're dealing with in other parts of the code. The usage is more clear when you look at references to the related `FieldExtensionComponent` type in the API reports, for example https://github.com/backstage/backstage/blob/da462fffc70688766cd70ab51c32d195b7074380/plugins/scaffolder/api-report.md#L239